Assets/Mirror/Editor/Weaver/Processors/RpcProcessor.cs

```diff
@@ -81,16 +87,19 @@ public static MethodDefinition ProcessRpcCall(TypeDefinition td, MethodDefinitio
                 rpcName = rpcName.Substring(RpcPrefix.Length);
             }
 
-            // invoke SendInternal and return
-            // this
+            int channel = clientRpcAttr.GetField("channel", 0);
+            bool excludeOwner = clientRpcAttr.GetField("excludeOwner", false);
+
+            // invoke SendInternal and return this
```

Comment:
```suggestion
            // invoke SendInternal and return
            // this
```
I Don't know what a better comment would be. The currently style in weaver seems to be comment of code generated followed by the create/Opcode  stuff